### PR TITLE
GitHub actions trial

### PR DESCRIPTION
**Pull request by robertparkinson**
### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,14 +9,16 @@
     "setup": "gulp sass copy-files",
     "start": "node server.js",
     "lint": "tsc --noEmit && tslint --project tsconfig.json && sass-lint -v -q",
-    "test": "TS_NODE_TRANSPILE_ONLY=true NODE_ENV=mocha LOG_LEVEL=OFF mocha --opts mocha.opts $(find src/test \\( -name '*.ts' \\) ! -path '*a11y*' ! -path '*/routes/*') --reporter-options reportFilename=unit,inlineAssets=true,reportTitle=citizen-frontend-unit",
+    "test": "echo 'unit tests are ran through GitHub actions as a mandatory step in order to save time on the main pipeline'",
```
Comment (KiranVarma02) [inline package.json]: Did you miss actual test here - ```TS_NODE_TRANSPILE_ONLY=true NODE_ENV=mocha LOG_LEVEL=OFF mocha --opts mocha.opts $(find src/test \\( -name '*.ts' \\) ! -path '*a11y*' ! -path '*/routes/*') --reporter-options reportFilename=unit,inlineAssets=true,reportTitle=citizen-frontend-unit"```

Comment (robertparkinson) [reply package.json]: The test:coverage runs all the unit and route tests so no need to duplicate them

Comment (KiranVarma02) [reply package.json]: there is no test:coverage after your changes isn't it? do you mean "test:action-coverage"? I would prefer test to run unit tests, coverage to run unit tests is misleading i think..

Comment (robertparkinson) [reply package.json]: the test coverage has been moved to the github action yes (test:action-coverage). we need to run the coverage test and as part of that test it runs the unit and route tests.

Comment (KiranVarma02) [reply package.json]: I don't like the idea of running coverage to run unit and route tests unless there is some valid reason. I see that PR is still WIP, it's probably something you can think about ..

Comment (robertparkinson) [reply package.json]: I've now created tests and tests:a11y so it is still similar to how we normally do it

**Pull request merged**